ref/general/auth-filter.adoc

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: You can use authentication filters to specify how Open Liberty processes authentication requests from different users and sources.
+:seo-title: Using authentication filters
+:seo-description: You can use authentication filters to specify how Open Liberty processes authentication requests from different users and sources.
+:page-layout: general-reference
+:page-type: general
+= Using authentication filters
+
+You can use authentication filters to specify how Open Liberty processes authentication requests from different users and sources.
+
+In applications where different kinds of protected resources are available to different users, authentication filters can control what types of authentication are available for which users. For example, in a banking application, a customer might authenticate by using social media credentials while an administrator must sign in with SPNEGO.
+
+Alternatively, you might want to secure a web application with the Social Login feature, while an associated microservice application is secured with MicroProfile JSON Web Token feature. In this case, you can configure an authentication filter so that the specified security configuration takes effect only when the conditions in the filter are met.
+
+To specify an authentication filter for a security feature, configure link:/docs/ref/config/#authFilter.html[the `authFilter` element] in your `server.xml` file. The `authFilter` element is available for the following Open Liberty features:
+
+* link:/docs/ref/feature/#jwtSso-1.0.html[JSON Web Token Single Sign-On]
+* link:/docs/ref/feature/#mpJwt-1.1.html[MicroProfile JSON Web Token]
+* link:/docs/ref/feature/#openid-2.0.html[OpenID]
+* link:/docs/ref/feature/#openidConnectClient-1.0.html[OpenID Connect Client]
+* link:/docs/ref/feature/#samlWeb-2.0.html[SAML web single sign-on]
+* link:/docs/ref/feature/#spnego-1.0.html[Simple and Protected GSSAPI Negotiation Mechanism (SPNEGO)]
+* link:/docs/ref/feature/#socialLogin-1.0.html[Social Login]
+
+== Authentication filter examples
+
+=== Request URL contains a pattern
+The following example shows a typical configuration for an authentication filter. In this configuration, any incoming request with a request URL that contains the `/SimpleServlet` pattern is processed by the service that is configured to use this filter.
```

Comment:
Do you think colons following the lead-ins to the examples would work better than periods? Maybe not, just a thought.

Reply:
Yeah- I was unsure about this. Either way is technically correct. My instinct says to use a period because in each case the lead in sentence (The following example...) is followed by a contextual sentence, which in some of these seems awkward with a colon. Used a period with all of them for consistency's sake